dotnet/SimpleSauce.Test/SauceOptionsTests.cs

```diff
@@ -107,12 +109,12 @@ public void AcceptsAllSauceLabsValues()
             SauceOptions.AvoidProxy.Should().BeTrue();
             SauceOptions.BuildName.Should().Be("Sample Build Name");
             SauceOptions.CapturePerformance.Should().BeTrue();
-            SauceOptions.ChromedriverVersion.Should().Be("71");
+            //SauceOptions.ChromedriverVersion.Should().Be("71");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/SauceLabs/sauce_bindings/pullRequest?prid=6342070)